src/python_routing_v02/compute_nhd_routing_SingleSeg_v02.py

```diff
@@ -250,6 +250,19 @@ def _handle_args():
         default="q_lateral",
     )
     parser.add_argument("--ql", help="QLat input data", dest="ql", default=None)
+
+    parser.add_argument(
+        "--data_assimilation_file_path",
+        help="Provide a path to a data assimilation routelink file folder",
+        dest="data_assimilation_parameters_file",
+        default=None,
+    )
+    parser.add_argument(
```

Comment:
- [ ] Make a new issue "Avoid option conflicts with data assimilation imports"
Ultimately, a folder+filter and csv don't make sense at the same time -- we could make them exclusive arguments or provide a log message/warning if the conflict is detected.